src/plugins/irc-events/link.js

```diff
@@ -103,21 +93,26 @@ function parse(msg, url, res, client) {
 		return;
 	}
 
-	emitToggle(client, toggle);
+	emitPreview(client, msg, preview);
 }
 
-function emitToggle(client, toggle) {
+function emitPreview(client, msg, preview) {
 	// If there is no title but there is preview or description, set title
 	// otherwise bail out and show no preview
-	if (!toggle.head.length) {
-		if (toggle.thumb.length || toggle.body.length) {
-			toggle.head = "Untitled page";
+	if (!preview.head.length && preview.type === "link") {
```

Comment:
For the record, this is the fix for that bug introduced by a previous PR: image previews were broken.